newIDE/app/src/CommandPalette/CommandHooks.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import * as React from 'react';
+import { type GlobalCommand } from './CommandManager';
+import CommandsContext from './CommandsContext';
+
+export const useGlobalCommand = (cmdName: string, cmdOpts: GlobalCommand) => {
+  const manager = React.useContext(CommandsContext);
```

Comment:
nitpicking: `const commandManager = `

(see other comments for justification).